Tests/conf.json

```diff
@@ -2297,7 +2297,7 @@
         "Windows Defender Advanced Threat Protection": "Test direct access to oproxy",
         "Microsoft Graph Groups": "Test direct access to oproxy",
         "SafeBreach": "SSL Issues",
-        "FeedOffice365": "SSl Issues. issue: 21181",
+        "Feed Office365": "SSl Issues. issue: 21181",
```

Comment:
What's the issue here?

Reply:
In the Mocks? I could find it prior to the initial merge.